app/presenters/curation_concerns/generic_work_show_presenter.rb

```diff
@@ -78,5 +78,11 @@ def show_thumb_member_presenters
         end
       end
     end
+
+    def riiif_file_id
+      return unless member_presenters.present?
+      representative_presenter = member_presenters.select{ |pres| pres.id == representative_id }.pop
+      representative_presenter.riiif_file_id
+    end
```

Comment:
If you want one thing, I think `find` is clearer (and slightly more efficient) than `select... pop`.